ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -5655,4 +5655,120 @@ RFVSC2X:
   - return_fan_run_status_2
   - return_fan_speed_percentage_command_2
   implements:
-  - OPERATIONAL
\ No newline at end of file
+  - OPERATIONAL
+
+EXTOR:
+  description: "shade control with extension monitoring based on outside illuminance"
+  is_abstract: true
+  uses:
+  - shade_extent_percentage_command
+  - outside_illuminance_sensor
```

Comment:
is there an active setpoint that it actively controls to? the setpoint limits below wont tell us what its actively trying to target.

Reply:
Based on the value of the outside illuminance sensor of the facade, the shade will be extended to either 100% when the outside illuminance exceeds the high_limit_illuminance_setpoint for a certain time, or the shade will pulled up if the outside illuminance falls below the low_limit_illuminance_setpoint. This is how it works in the "Auto-Mode". The user can overrun the auto-command manually by activating the wall mounted controls and drive the shade to any given position. After two hours, the shade will go back to "Auto-Mode"